beams/hooks.py

```diff
@@ -314,5 +314,11 @@
     ]},
     {"dt": "Workflow Action Master", "filters": [
         ["name", "in", ["Submit for Approval","Reopen", "Approve", "Reject", "Send For Finance Verification", "Verify", "Send for Approval","Submit","Cancel","Send Email To Party","Put On Hold","Assign to Admin","Assign to Enquiry Officer","Start Enquiry","Start Enquiry"]]
+    ]},
+    {"dt": "Role", "filters": [
+        ["name", "in", ["CEO","Production Manager","Company Secretary","Hod"]]
+    ]},
+    {"dt": "Email Template", "filters": [
+        ["name", "in", ["Job Applicant Follow Up"]]
```

Comment:
the role fixture was removed, do not add it back, as well as adding the template in setup.py instead of fixtures